src/randoop/sequence/ExecutableSequence.java

```diff
@@ -392,6 +391,10 @@ public static boolean getRuntimeInputs(Sequence s, List<ExecutionOutcome> outcom
     return true;
   }
   
+  public static Object[] getRuntimeValuesForVars(List<Variable> vars, Execution execution) {
```

Comment:
What is the format for the variables?  (More generally, please add documentation.)